### ISSUE TITLE
Allow users to see their ETDs pre-graduation

### ISSUE THREAD

**Issue by bess** [curationexperts/laevigata#1433]
- [ ] After I've submitted my work, and before I've graduated, I should be able to ~search for my ETD and find it~ see my work under My Works
- [x] There should be a menu item in the top right corner taking me to My ETD (the My Works dashboard)

**Comment by bess** [comment]
My Works has been fixed in Hyrax master. We need to either incorporate that fix or (preferably) get it merged into Hyrax and get a Hyrax point release out

**Comment by bess** [comment]
Waiting on release of Hyrax 2.1.1
